spec.emu

```diff
@@ -343,30 +382,158 @@ contributors: Ron Buckton, Ecma International
-      </ins>
+        <ins>ThrowExpression[?Yield, ?Await]</ins>
     </emu-grammar>
+
+    <ins class="block">
+    <emu-clause id="sec-throw-operator">
+      <h1><ins>The `throw` Operator</ins></h1>
+      <h2>Syntax</h2>
+      <emu-grammar type="definition">
+        ThrowExpression[Yield, Await] :
+          `throw` UnaryExpression[?Yield, ?Await] [lookahead ∉ ThrowExpressionInvalidPunctuator]
+
+        ThrowExpressionInvalidPunctuator : one of `,` `&lt;` `>` `&lt;=` `>=` `==` `!=` `===` `!==` `*` `%` `**` `&lt;&lt;` `>>` `>>>` `&` `|` `^` `&&` `||` `??` `?`
+      </emu-grammar>
+
+      <emu-clause id="sec-throw-operator-runtime-semantics-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+        <emu-grammar>ThrowExpression : `throw` UnaryExpression</emu-grammar>
+        <emu-alg>
+          1. Let _exprRef_ be ? Evaluation of |UnaryExpression|.
+          1. Let _exprValue_ be ? GetValue(_exprRef_).
+          1. Return ThrowCompletion(_exprValue_).
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+    </ins>
   </emu-clause>
 
-  <emu-clause id="sec-throw-operator">
-    <h1><ins>The `throw` Operator</ins></h1>
-    <emu-clause id="sec-throw-operator-runtime-semantics-evaluation">
-      <h1>Runtime Semantics: Evaluation</h1>
-      <emu-grammar>UnaryExpression : `throw` UnaryExpression</emu-grammar>
-      <emu-alg>
-        1. Let _exprRef_ be ? Evaluation of |UnaryExpression|.
-        1. Let _exprValue_ be ? GetValue(_exprRef_).
-        1. Return ThrowCompletion(_exprValue_).
-      </emu-alg>
+  <emu-clause id="sec-multiplicative-operators">
+    <h1>Multiplicative Operators</h1>
+    <ins class="block">
+    <emu-clause id="sec-multiplicative-operators-static-semantics">
+      <h1>Static Semantics</h1>
+      <emu-clause id="sec-multiplicative-operators-static-semantics-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          MultiplicativeExpression :
+            MultiplicativeExpression MultiplicativeOperator ExponentiationExpression
+        </emu-grammar>
+        <ul>
+          <li>
+            It is a Syntax Error if ContainsThrowOnRight of |MultiplicativeExpression| is *true*.
+          </li>
+        </ul>
+        <emu-note>
+          <p>This production exists in order to prevent automatic semicolon insertion rules (<emu-xref href="#sec-automatic-semicolon-insertion"></emu-xref>) from being applied to the following code:</p>
+          <pre><code class="javascript">
+            a ?? throw b
+            /c/d
+          </code></pre>
+          <p>so that it would be interpreted as two valid statements. The purpose is to maintain consistency with similar code using |ThrowStatement|:</p>

```

Comment:
Possibly, though I'd like an editor to weigh in as I think "maintain consistency" here and in _OptionalChain TemplateExpression_ refers specifically to ASI handling, since both grammars are intended to produce an error.